go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/open-cluster-management/governance-policy-propagator
+module github.com/stolostron/governance-policy-propagator
```

Comment:
`github.com/open-cluster-management/api v0.0.0-20200610161514-939cead3902c` is not replaced